atlasdb-api/src/main/java/com/palantir/atlasdb/encoding/PtBytes.java

```diff
@@ -67,56 +68,56 @@ public static long toLong(byte[] bytes, int offset) {
     /**
      * Converts a string to a UTF-8 byte array.
      */
-    public static byte[] toBytes(String s) {
-        return s.getBytes(Charsets.UTF_8);
+    public static byte[] toBytes(String str) {
+        return str.getBytes(StandardCharsets.UTF_8);
     }
 
     /**
      * Converts a string to a UTF-8 byte array. This method will cache results and return cached
      * results, so the resulting byte arrays must not be modified.
      */
-    public static byte[] toCachedBytes(String s) {
-        byte[] bytes = cache.getIfPresent(s);
+    public static byte[] toCachedBytes(String str) {
```

Comment:
Well, in this case, `arr` and `a` are both just as understandable.

It's mainly for the case where you have a long method and you use single letter variable names, or you start writing a method, call the variables `a` and `b`, then finish the method and forget to think of reasonable names for the variables.

This doesn't really apply to short methods, but I wouldn't disable the rule because of that. It's not that hard for short methods to use `arr` instead of `a` or even to just make it more explicit with something like `arrayToEncode`.